task/common/machine/testdata/machine_script_minimal.golden

```diff
@@ -48,7 +48,7 @@ sudo tee /etc/systemd/system/tpi-task.service > /dev/null <<END
 [Service]
   Type=simple
   ExecStart=-$TPI_START_COMMAND
-  ExecStop=/bin/bash -c 'source /opt/task/credentials; systemctl is-system-running | grep stopping || echo "{\\\\"result\\\\": \\\\"\$SERVICE_RESULT\\\\", \\\\"code\\\\": \\\\"\$EXIT_STATUS\\\\", \\\\"status\\\\": \\\\"\$EXIT_CODE\\\\"}" > "$TPI_LOG_DIRECTORY/status-$TPI_MACHINE_IDENTITY" && RCLONE_CONFIG= rclone copy "$TPI_LOG_DIRECTORY" "\$RCLONE_REMOTE/reports"'
+  ExecStop=/bin/bash -c 'source /opt/task/variables; source /opt/task/credentials; if systemctl is-system-running | grep stopping; then status=queued; else echo "{\\\\"result\\\\": \\\\"\$SERVICE_RESULT\\\\", \\\\"code\\\\": \\\\"\$EXIT_STATUS\\\\", \\\\"status\\\\": \\\\"\$EXIT_CODE\\\\"}" > "$TPI_LOG_DIRECTORY/status-$TPI_MACHINE_IDENTITY" && RCLONE_CONFIG= rclone copy "$TPI_LOG_DIRECTORY" "\$RCLONE_REMOTE/reports"; if test $SERVICE_RESULT == timeout; then status=timeout; else; test $EXIT_STATUS == 0 && status=succeeded || status=failed; fi; fi && leo report --type=data --url $STUDIO_URL --token $STUDIO_TOKEN --repo $STUDIO_REPO_URL --param status=$status'
```

Comment:
Shouldn't `\\\\"` be `\\\"`?

Reply:
I don't recall exactly why, but this is the right amount of backslashes and it works. 😅

Reply:
They should never be an odd number, though.

Reply:
Going through levels of escaping:
`"` -> `\`+`"` -> `\\` + `\"`

Reply:
Either way, if we're having discussions about how many backslashes we need to escape this, they are beside the point.